tests/backup/backup_test.go

```diff
@@ -191,81 +194,153 @@ var _ = Describe("{BasicBackupCreation}", func() {
 		})
 
 		Step("Creating rules for backup", func() {
+			var err error
 			log.InfoD("Creating pre rule for deployed apps")
-			pre_rule_status, pre_rule_uid = CreateRuleForBackup("backup-pre-rule", "default", app_list, "pre", ps)
-			dash.VerifyFatal(pre_rule_status, true, "Verifying pre rule for backup")
+			preRuleStatus, preRuleUid, err = Inst().Backup.CreateRuleForBackup(PreRuleName, "default", appList,
+				"pre", ps)
+			if err != nil {
```

Comment:
err!=nil check is not required FailOnError will take care of it

Reply:
done!!